frameworks/cassandra/tests/test_custom_domain.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+import pytest
+
+import sdk_hosts
+import sdk_install
+import sdk_networks
+
+from tests import config
+
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="module", autouse=True)
+def configure_package(configure_security):
+    try:
+        sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+
+        yield  # let the test session execute
+    finally:
+        sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+
+
+@pytest.mark.sanity
+def test_custom_domain():
+    task_count = 3
+    custom_domain = sdk_hosts.get_crypto_id_domain()
+    sdk_install.install(
+        config.PACKAGE_NAME,
+        config.SERVICE_NAME,
+        task_count,
+        additional_options={"service": {"security": {"custom_domain": custom_domain, }, }, }
```

Comment:
if this is json you should remove trailing `,`s

Reply:
Fixed